etc/rc1

```diff
@@ -30,7 +30,10 @@ modload 0 cron sync=heartbeat.pulse
 modload 0 job-manager
 modload all job-info
 modload 0 job-list
-modload 0 job-archive
+period=`flux config get --default= archive.period`
+if test $RANK -eq 0 -a -n "${period}"; then
+    modload 0 job-archive
```

Comment:
Since the rank has already been checked (and that's all `modload()` does), probably better to just call `flux module load job-archive` here.